core/AutocorrectSuggestion.cc

```diff
@@ -26,6 +26,32 @@ bool hasSeen(const UnorderedSet<Loc> &seen, Loc loc) {
     return false;
 }
 
+string AutocorrectSuggestion::unsafeApplyToString(const std::string_view source) {
+    UnorderedSet<Loc> seen;
+    vector<AutocorrectSuggestion::Edit> edits = move(this->edits);
```

Comment:
Can we just duplicate `this->edits` here, rather than mutating `this`?  (Bonus points for making this method `const`, I guess.)  It's not immediately obvious to me that `unsafeX` would mutate the object.